cmd/provider/main.go

```diff
@@ -53,7 +54,11 @@ func main() {
 	cfg, err := ctrl.GetConfig()
 	kingpin.FatalIfError(err, "Cannot get API server rest config")
 
-	mgr, err := ctrl.NewManager(cfg, ctrl.Options{SyncPeriod: syncPeriod})
+	mgr, err := ctrl.NewManager(cfg, ctrl.Options{
+		LeaderElection:   *leaderElection,
+		LeaderElectionID: "crossplane-leader-election-provider-aws",
```

Comment:
@negz  @hasheddan I believe it is possible to install multiple versions of a same provider at the same time, right? Is it possible for them to end up in the same namespace? If yes, this line will cause a collision.

Reply:
This could only be possible if someone tried to make it happen (i.e. installed the same provider twice under different names) and none of the CRDs overlapped (a revision would fail to start if there were any CRDs in conflict), so we are safe from conflict here 👍

Reply:
Ah good to know, thanks! I will update the description of this and other PRs for test results and they should be good to go.